pyproject.toml

```diff
@@ -17,9 +16,17 @@ python-dotenv = "^0.20.0"
 psycopg2 = {version = "^2.9.3", optional = true, extras = ["cockroach_db"]}
 cryptography = {version = "^37.0.4", optional = true, extras = ["cockroach_db", "mysql"]}
 PyMySQL = {version = "^1.0.2", optional = true, extras = ["mysql"]}
+mysql = "^0.0.3"
```

Comment:
Ada sebab ke mengapa pakej `mysql` diperlukan sedangkan sudah ada `PyMySQL`? Ada kelebihan ke?

(`PyMySQL` ialah sejenis optional dependency yang boleh diakses menggunakan `poetry install --extras mysql`)

Reply:
Kalau saya jalan `Extra [mysql] is not specified.`, akan dapat error berikut:

`Extra [mysql] is not specified.`

Tapi ini untuk kegunaan di cabang saya. Kalau tidak diperlukan, saya keluarkan dari PR.

Reply:
Ok pelik. Tidak mengapa. Biar dulu. Nanti aku tengok macam mana.